deepchecks/tabular/checks/distribution/train_test_feature_drift.py

```diff
@@ -242,24 +253,26 @@ def condition(result: Dict) -> ConditionResult:
                                                          reverse=True)]
             columns_to_consider = columns_to_consider[:number_of_top_features_to_consider]
             not_passing_categorical_columns = {column: f'{d["Drift score"]:.2}' for column, d in result.items() if
-                                               d['Drift score'] > max_allowed_psi_score and d['Method'] == 'PSI'
+                                               d['Drift score'] > max_allowed_categorical_score and
+                                               d['Method'] in SUPPORTED_CATEGORICAL_METHODS
                                                and column in columns_to_consider}
             not_passing_numeric_columns = {column: f'{d["Drift score"]:.2}' for column, d in result.items() if
-                                           d['Drift score'] > max_allowed_earth_movers_score
-                                           and d['Method'] == "Earth Mover's Distance"
+                                           d['Drift score'] > max_allowed_numerical_score
+                                           and d['Method'] in SUPPORTED_NUMERICAL_METHODS
                                            and column in columns_to_consider}
             return_str = ''
             if not_passing_categorical_columns:
-                return_str += f'Found categorical columns with PSI above threshold: {not_passing_categorical_columns}\n'
+                return_str += 'Found categorical columns with drift score above threshold: ' \
+                              f'{not_passing_categorical_columns}\n'
             if not_passing_numeric_columns:
-                return_str += f'Found numeric columns with Earth Mover\'s Distance above threshold: ' \
+                return_str += f'Found numerical columns with drift score above threshold: ' \
                               f'{not_passing_numeric_columns}'
 
             if return_str:
                 return ConditionResult(ConditionCategory.FAIL, return_str)
             else:
                 return ConditionResult(ConditionCategory.PASS)
 
-        return self.add_condition(f'PSI <= {max_allowed_psi_score} and Earth Mover\'s Distance <= '
-                                  f'{max_allowed_earth_movers_score}',
+        return self.add_condition(f'categorical drift score <= {max_allowed_categorical_score} and '
```

Comment:
Use the method's name. In general, even if I miss this in this PR, do not use "categorical/numeric drift score".

Reply:
I don't know which methods will be used in the check

Reply:
Okay, right. So let's keep it like that, but make sure that in the return_str of the condition it mentions the methods